Levi9.ERP/Datas/Requests/Currency.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace Levi9.ERP.Datas.Requests
+{
+    public enum Currency
```

Comment:
CurrencyType from models should be used. Multiple of the same enum classes should not be created.